src/stylesheets/components/_np-score.scss

```
@@ -0,0 +1,76 @@
+.np-score {
+  margin-top: rem(45px);
+
+  text-align: center;
+
+  &__number-container {
+    width: fit-content;
+    height: rem(56px);
+    margin: 0 auto;
+    padding: 0;
+
+    list-style: none;
+
+    border: 0.5px solid map-get($brand-white, 'primary');
+    border-radius: rem(12px);
+  }
+
+  &__number-label {
+    display: inline-block;
+    height: rem(55px);
+    margin: 0;
+
+    border-right: 0.5px solid map-get($brand-white, 'primary');
+  }
+
+  &__number-label--no-border {
+    border-right: 0;
+  }
+
+  &__number {
+    margin: rem(15px 13px);
+
+    color: map-get($brand-white, 'primary');
+    font-weight: map-get($font-weight, 'bolder');
+    font-size: rem(20px);
+    line-height: rem(25px);
+
+    cursor: pointer;
+    opacity: 0.5;
+
+    transition: color 200ms;
+  }
+
+  &__number-container:hover > &__number-label &__number {
+    opacity: 1;
+  }
+
+  &__number-container > &__number-label:hover ~ &__number-label &__number { /* stylelint-disable-line */
+    opacity: 0.5;
+  }
+
+  &__number--selected {
+    opacity: 1;
+  }
+
+  &__identifier {
+    @include make-row();
+
+    margin-top: rem(15px);
+    padding: rem(0 15px);
+  }
+
+  &__low-score {
+    @include make-col(6);
+
+    color: map-get($brand-white, 'primary');
+    text-align: start;
+  }
+
+  &__high-score {
+    @include make-col(6);
+
+    color: map-get($brand-white, 'primary');
+    text-align: end;
+  }
```

@rafayet-monon How about we group it like this?

```suggestion
  &__low-score,
  &__high-score {
    @include make-col(6);

    color: map-get($brand-white, 'primary');
  }

  &__low-score {
    text-align: start;
  }

  &__high-score {
    text-align: end;
  }
```